### PR TITLE
1545 show sow history

### PR DESCRIPTION
**Pull request by marcellmueller**
- feat: enable history tracking on sow and project information data tables
- feat: return project information history in computed column
- feat: add project information updates to history page

Implements # \<issue number\>

### CODE REVIEW

**Review by VB-at-Bis — CHANGES_REQUESTED** [review]
Please see my comments

File: db/sqitch.plan [hunk 1]
```diff
@@ -322,3 +322,5 @@ tables/sow_tab_1 2023-05-29T14:44:45Z Anthony Bushara <anthony@button.is> # Tabl
 tables/project_information_data 2023-05-25T20:56:37Z Marcel Mueller <marcel@m2> # form data for project information form
 mutations/create_project_information 2023-05-25T21:27:43Z Marcel Mueller <marcel@m2> # add mutation create project information
 computed_columns/application_project_information 2023-05-25T23:41:42Z Marcel Mueller <marcel@m2> # add computed column project_information
+extensions/audit_sow.sql 2023-06-01T21:52:15Z Marcel Mueller <marcel@m2> # enable history tracking for sow and project information
```
[the remark this answers is not in the document]
Comment (marcellmueller) [reply db/sqitch.plan]: Thanks, fixed.

File: db/deploy/extensions/audit_sow.sql [hunk 2]
```diff
@@ -0,0 +1,8 @@
+-- Deploy ccbc:extensions/audit_sow.sql to pg
+
+begin;
+
+select audit.enable_tracking('ccbc_public.application_sow_data'::regclass);
+select audit.enable_tracking('ccbc_public.project_information_data'::regclass);
```
Comment (VB-at-Bis) [inline db/deploy/extensions/audit_sow.sql:6]: We have that already in db\deploy\tables\project_information_data.sql

**Review by VB-at-Bis — CHANGES_REQUESTED** [review]
No need to enable audit for project_information_data - it is already enabled

**Review by VB-at-Bis — APPROVED** [review]
Looks good!

**Pull request merged**